SpywareFilter/sections/specific.txt

```diff
@@ -51,6 +51,7 @@
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp
+/wp-power-stats.js$domain=ncrc.org|firmasport.ee|lemire.me|prostreaming.net|musicalnews.com
```

Comment:
I think it can be blocked without domain restriction
https://publicwww.com/websites/%22%2Fwp-power-stats.js%22/